public/index.js

```diff
@@ -139,14 +148,42 @@
     element.style.backgroundPosition = parseInt(value) * illustrationSize + "px";
   }
 
-  function addOnInputToElement(element, func){
-    element.oninput = function(){ func() }
-  }
-
   function emojiSprite(){
     var value = document.getElementsByClassName("range")[0].value;
     var element = document.getElementsByClassName("friends__emoji-sprite")[0];
     changeBackgroundPosition(element, value, -180)
   }
 
+  function emailValidator(emailAddress){
+    var regex = RegExp(
+      '^(([^<>()\\[\\]\\\\.,;:\\s@"]+(\\.[^<>()\\[\\]\\\\.,;:\\s@"]+)*)|(".' +
+      '+"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-z' +
+      'A-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$'
+    )
+    return regex.test(emailAddress) ? true : false;
+  }
+
+  function sendMail(){
+    var emailRecipient = document.getElementsByClassName("finish__email-input")[0];
+
+    if (!emailValidator(emailRecipient.value)){
+      emailRecipient.value = 'Please enter a valid email address.'
+      return;
+    }
+
+    var mailOptions = {
```

Comment:
When you add your email address on the last page it goes into the mailOptions object, we then send this object in a post request to the backend where the sendMail function is invoked. I could just send the email address in the post request, would this be better?